feat: DIA-1579: Seeded Sample Project and Prompts for Trials

robot-ci-heartex opened the pull request:
Hi @robot-ci-heartex!

This PR was automaticaly generated via Follow Merge. 
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.

- https://github.com/HumanSignal/label-studio-sdk/pull/351 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/11664005377)

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11664005377

Commit:
Add user.get_token() helper

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11664429119

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11664633625

matt-bernstein commented on label_studio/users/models.py:
```diff
@@ -177,6 +177,9 @@ def get_full_name(self):
     def get_short_name(self):
         """Return the short name for the user."""
         return self.first_name
+    
+    def get_token(self) -> Token:
+        return Token.objects.filter(user=self).first()
```
I wonder if there's a reason this doesn't already exist - should the token always be reset instead of read when it is used programmatically? If there isn't, lgtm

niklub replied:
no, our tokens are long living

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11741280788

niklub committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11741462570

robot-ci-heartex committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11741476823

Commit:
fmt
